ironfish-rust/src/transaction/mod.rs

```diff
@@ -220,6 +223,28 @@ impl ProposedTransaction {
         self._partial_post()
     }
 
+    fn _change_note_amounts(&self, asset_id: &[u8; 32], change_amount: &i64) -> Vec<i64> {
```

Comment:
I do think it would make sense if we are going to do this to make it more parameterized either by passing in the threshold for splitting notes or passing in a list of change amounts that we want. That way the wallet can make the decision of how to create the transaction instead of the rust code doing it automatically